src/current/_includes/v23.1/ui/sessions.md

```diff
@@ -73,6 +73,16 @@ The **Cancel session** button ends the session. The client that holds this sessi
 
   A list of the most recent transaction fingerprint IDs, represented in hexadecimal, executed by this session. The fingerprint ID is a link to the [Transaction Details page]({{ link_prefix }}ui-transactions-page.html#transaction-details-page) for the transaction. When you click the link, in order to fetch and render the transaction details, the global date range is changed to the session's start and end time.
 
+### Closed session cache
+
+Details about closed sessions are stored in memory in a closed session cache. This cache is controlled by the following two [cluster settings]({{ link_prefix }}cluster-settings.html):
```

Comment:
Is 'closed sessions' referring to "sessions" as in `SHOW SESSIONS`?  Or something else that is specific to the DB Console? 

If the former, maybe it would be useful somehow to also link to the `SHOW SESSIONS` SQL docs (https://www.cockroachlabs.com/docs/stable/show-sessions) so users know about that additional tool.  Leave it to your discretion, maybe not useful here.

Reply:
The link to `SHOW SESSIONS` is already in the [**See also** section](https://deploy-preview-18107--cockroachdb-docs.netlify.app/docs/v23.1/ui-sessions-page.html#see-also), so I will not add any other additional link.